src/shared/seccomp-util.c

```diff
@@ -1310,7 +1367,11 @@ int seccomp_parse_syscall_filter(
         } else {
                 int id;
 
-                id = seccomp_syscall_resolve_name(name);
+                r = dlopen_libseccomp();
+                if (r < 0)
+                        return r;
```

Comment:
Ah, more simpler solution is gracefully ignore the failure, as this is just a parser of user input.